### PR TITLE
feat(react): add FormSG icons, title and description

### LINKED ISSUE #1901: [React] Set up base app icons, title, description.
The current base app in `form-v2/develop` is still using Create React App defaults. We should change them to the desired favicons, page description and titles for consistency.

The files to look at are
- `/frontend/public/manifest.json`
- `/frontend/public/favicon.ico`
- `/frontend/public/logo192.png`
- `/frontend/public/logo512.png`
- `/frontend/public/index.html`

May want to look at some articles like https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs for what needs to be done

### PR DESCRIPTION
**Pull request by chowyiyin**
This PR replaces the default icons, title and description of the React base app with those for FormSG.

Closes #1901

## Solution
Files added:
- `favicon.ico`: 32x32 and 16x16
- `favicon.svg`
- `logo192.png`: 192x192
- `logo512.png`: 512x512
- `apple-touch-icon.png`: 180x180
Used `Squoosh` and `SVGO` to optimize `svg` and `png`s as suggested in issue's linked article

## Screenshots
**On desktop browser**
Screenshot 2021-06-03 at 11 34 36 AM

**On iPhone**
photo6179124987709992202
photo6179124987709992201

**On Android device**
Note: `logo192.png` used for homescreen; `logo512.png` and `name` in `manifest.json` used for splash screen.
Screenshot 2021-06-03 at 11 39 02 AM
Screenshot 2021-06-03 at 11 37 50 AM
Screenshot 2021-06-03 at 11 50 09 AM

## Manual Tests
- [ ] Open FormSG on different desktop browsers. Should have FormSG icon displayed on tab.
- [ ] Open FormSG on other Apple and Android devices. Add to home screen. FormSG icon should be displayed on web clip, on app in home screen and splash screen (for Android only)

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -2,13 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/logomark.ico" />
```
Comment (karrui) [inline frontend/public/index.html:5]: can we continue to call it favicon? that's the normal name for... well, the favicon.

File: frontend/public/index.html [hunk 2]
```diff
@@ -2,13 +2,10 @@
 <html lang="en">
```
Comment (karrui) [inline frontend/public/index.html:2]: Can we also generate the correct files that are usually used for favicons on the web?

See the linked article in the original issue: https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs

**Review by karrui — CHANGES_REQUESTED** [review]

File: frontend/public/index.html (hunk 2 above)
Comment (chowyiyin) [reply frontend/public/index.html:2]: May I clarify what you mean by correct files? Am I required to follow all the steps in the linked article or do I just have to generate the following files:
`logo192.png`
`logo512.png`
`apple-touch-icon.png`
`favicon.ico`
`favicon.svg`
Do let me know if I missed out anything thanks!

Comment (karrui) [reply frontend/public/index.html:2]: Yeah, just generate the files; ensure they are the correct size or minified etc. The article is just a guide

**Review by karrui — COMMENTED** [review]
the favicon.ico does not seem to be transparent and have a white background instead, should we revert it to follow favicon.svg? Maybe use https://realfavicongenerator.net/ for generating the necessary files

**Comment by karrui** [comment]
Closing this to reduce clutter. @chowyiyin pls reopen when you pick this up again.

**Comment by karrui** [comment]
Reopening this, React issues are a go!

**Review by karrui — COMMENTED** [review]
favicon.ico seems to be quite large, is there a way to minimize the size?

**Review by karrui — APPROVED** [review]
just realized our current logomark is 100kb... omg....

**Pull request merged**